tests/upgrade/integration/generated/post/query - diddoc/diddoc_indy_16.json

```diff
@@ -13,19 +13,19 @@
                 "id": "did:cheqd:testnet:5rjaLzcffhGUH4nt4fyfAg#key-1",
                 "verificationMethodType": "Ed25519VerificationKey2020",
                 "controller": "did:cheqd:testnet:5rjaLzcffhGUH4nt4fyfAg",
-                "verification_material": "{\"publicKeyMultibase\":\"zBNy7xYrzNb25mbVFP5hVZXwJZ7RD16yJSoya82qe3Jra\"}"
+                "verification_material": "z6MkpqEAYo7Ri8WYt6Kx4efLQdVJNgh4QzDf8ptVxJoexXdx"
             },
             {
                 "id": "did:cheqd:testnet:5rjaLzcffhGUH4nt4fyfAg#key-2",
                 "verificationMethodType": "Ed25519VerificationKey2020",
                 "controller": "did:cheqd:testnet:5rjaLzcffhGUH4nt4fyfAg",
-                "verification_material": "{\"publicKeyMultibase\":\"zC4mnpocVYbywi32srkHzRdy7LEXQG4TM8YtCTPXWnza1\"}"
+                "verification_material": "z6MkqX2qR3rvt9UQpXsaYKFqGjX79ooFfwhhpZo8HfVXiDMP"
```

Comment:
This is the same throughout, but on decoding it doesn't seem it says `0xed01` in hex?